dpnp/dpnp_iface_statistics.py

```diff
@@ -352,69 +350,100 @@ def histogram(a, bins=10, range=None, density=None, weights=None):
     )
 
 
-def max(x1, axis=None, out=None, keepdims=False, initial=None, where=True):
+def max(a, axis=None, out=None, keepdims=False, initial=None, where=True):
     """
     Return the maximum of an array or maximum along an axis.
 
+    For full documentation refer to :obj:`numpy.max`.
+
+    Returns
+    -------
+    out : dpnp.ndarray
+        Maximum of `a`.
+
     Limitations
     -----------
-    Input array is supported as :obj:`dpnp.ndarray`.
-    Otherwise the function will be executed sequentially on CPU.
-    Parameter `out` is supported only with default value ``None``.
+    Input and output arrays are only supported as either :class:`dpnp.ndarray` or :class:`dpctl.tensor.usm_ndarray`.
```

Comment:
otherwise the line is too long
```suggestion
    Input and output arrays are only supported as either :class:`dpnp.ndarray`
    or :class:`dpctl.tensor.usm_ndarray`.
```